README.md

```diff
@@ -150,6 +152,20 @@ a playbook and executed before the test run, or a single shell command which
 will be executed using the Ansible `raw` module (so the two examples above are
 exactly equivalent).
 
+Use the `min_ansible_version` key to indicate the minimum version of
+ansible that can be used.  For example, if you cannot use ansible 2.7
+to manage Fedora 32 hosts, use something like this:
+```json
+    {
+      "name": "fedora-32",
+      "source": "http://download.fedoraproject.org/pub/fedora/linux/releases/32/Cloud/x86_64/images/Fedora-Cloud-Base-32-1.6.x86_64.qcow2",
+      "upload_results": false,
```

Comment:
This does not need to be false, does it? Also `false` should be the default, so this seems to be better:
```suggestion
      "upload_results": true,
```

Reply:
https://github.com/linux-system-roles/test-harness/pull/87/commits/a1f7e36ec876eb927bb6ddcd25ba619b13dc88c0